doc/classes/EditorSettings.xml

```diff
@@ -1380,7 +1380,11 @@
 			If [code]true[/code], uses [StringName] instead of [String] when appropriate for code autocompletion.
 		</member>
 		<member name="text_editor/completion/add_type_hints" type="bool" setter="" getter="">
-			If [code]true[/code], adds [url=$DOCS_URL/tutorials/scripting/gdscript/static_typing.html]GDScript static typing[/url] hints such as [code]-&gt; void[/code] and [code]: int[/code] when using code autocompletion or when creating onready variables by drag and dropping nodes into the script editor while pressing the [kbd]Ctrl[/kbd] key. If [code]true[/code], newly created scripts will also automatically have type hints added to their method parameters and return types.
+			If [code]true[/code], automatically adds [url=$DOCS_URL/tutorials/scripting/gdscript/static_typing.html]GDScript static typing[/url] (such as [code]-&gt; void[/code] and [code]: int[/code]) in many situations where it's possible to, including when:
+			- Accepting a suggestion from code autocompletion;
+			- Creating a new script from a template;
+			- Connecting signals from the Node dock;
+			- Creating variables prefixed with [annotation @GDScript.@onready], by dropping nodes from the Scene dock into the script editor while holding [kbd]Ctrl[/kbd].
```

Comment:
```suggestion
			- Creating variables prefixed with [annotation @GDScript.@onready], by dropping nodes from the Scene dock into the script editor while holding the [kbd]Ctrl[/kbd].
```

Reply:
No, that's not right. Either `the [kbd]Ctrl[/kbd] key` or `[kbd]Ctrl[/kbd]` standalone. I chose the latter because it's actually noticeably more common in the class reference. That's possibly because the `[kbd]` tag already implies it, or it's more akin to how some applications describe actions for brevity ("Press <kbd>Space</kbd> to jump!").

Reply:
Missed the missing `key`